chain-abci/src/storage/mod.rs

```diff
@@ -15,9 +15,9 @@ use enclave_protocol::{IntraEnclaveRequest, IntraEnclaveResponseOk, SealedLog};
 /// spend_utxo: Modify UTxO storage
 /// create_utxo: Write into UTxO storage
 /// sealed_log: Write into storage
-/// deposit/withdraw: Modify staking state and related validator state structure
+/// deposit/withdraw/unbond: Modify staking state and related validator state structure
 #[derive(Debug, Clone)]
-pub enum TxEnclaveAction {
+pub enum TxAction {
```

Comment:
This modifications changed the intension of `TxEnclaveAction` and makes `execute_enclave_tx` have to panic in one branch.
Maybe a wrapping type would be better:
```
enum TxAction {
  Enclave(TxEnclaveAction),
  Public(TxPublicAction),
}

Reply:
Ok! I will update it